haskell-src/Concordium/Types/Parameters.hs

```diff
@@ -1771,7 +1787,10 @@ data ChainParameters' (cpv :: ChainParametersVersion) = ChainParameters
       -- | The finalization committee parameters.
       --  These parameters are introduced as part of protocol 6 (cpv2).
       --  The set of parameters shares the 'Authorization' with the '_cpPoolParameters'.
-      _cpFinalizationCommitteeParameters :: !(OParam 'PTFinalizationCommitteeParameters cpv FinalizationCommitteeParameters)
+      _cpFinalizationCommitteeParameters :: !(OParam 'PTFinalizationCommitteeParameters cpv FinalizationCommitteeParameters),
+      -- | The score parameters.
+      --  These parameters are introduced as part of protocol 8 (cpv3).
+      _cpMaxScore :: !(OParam 'PTScoreParameters cpv ScoreParameters)
```

Comment:
Suggested renaming:
```suggestion
      _cpValidatorScoreParameters :: !(OParam 'PTValidatorScoreParameters cpv ValidatorScoreParameters)
```

Reply:
Or possibly it should be something like ValidatorSuspensionParameters.

Reply:
I like `ValidatorScoreParameters` because we keep a door open to maybe do something different with the score in the future.